QuietPlace/App.js

```diff
@@ -43,4 +42,16 @@ const styles = StyleSheet.create({
       alignItems: 'center',
       justifyContent: 'center',
    },
+   homelogo: {
```

Comment:
Home logo I think could be placed further up, there is too much space from the top of my phone (S10+) to the logo.